docs/source/tutorial/trainer.rst

```diff
@@ -0,0 +1,340 @@
+
+:class:`~chainer.Reporter` aggregates the results to output to the standard
+output. The timing for displaying the output can be given by the list.
+
+:class:`~chainer.training.extensions.PlotReport`
+................................................
+
+:class:`~chainer.training.extensions.PlotReport` plots the values specified by
+its arguments, draws the graph and saves the image in the directory set by
+:attr:`~chainer.training.extensions.PlotReport.file name`.
+
+The :class:`~chainer.training.Extension` class has a lot of options other than
+those mentioned here. For instance, by using the
+:attr:`~chainer.training.Extension.trigger` option, you can set individual
+timings to activate the :class:`~chainer.training.Extension` more flexibly.
+Please take a look at the official document in more detail：
+`Trainer extensions`_<reference/extensions.html>.
+
+7. Start Training
+'''''''''''''''''
+
+To start training, just call :meth:`~chainer.training.Trainer.run` method from
+:class:`~chainer.training.Trainer` object.
+
+.. testcode::
+
+    trainer.run()
+
+::
+
+    epoch       main/loss   main/accuracy  validation/main/loss  validation/main/accuracy  elapsed_time
+    1           1.53241     0.638409       0.74935               0.835839                  4.93409
+    2           0.578334    0.858059       0.444722              0.882812                  7.72883
+    3           0.418569    0.886844       0.364943              0.899229                  10.4229
+    4           0.362342    0.899089       0.327569              0.905558                  13.148
+    5           0.331067    0.906517       0.304399              0.911788                  15.846
+    6           0.309019    0.911964       0.288295              0.917722                  18.5395
+    7           0.292312    0.916128       0.272073              0.921776                  21.2173
+    8           0.278291    0.92059        0.261351              0.923457                  23.9211
+    9           0.266266    0.923541       0.253195              0.927314                  26.6612
+    10          0.255489    0.926739       0.242415              0.929094                  29.466
+
+Let's see the graph of loss saved in the ``mnist_result`` directory.
+
+.. image:: ../../image/trainer/mnist_loss.png
+
+How about the accuracy?
+
+.. image:: ../../image/trainer/mnist_accuracy.png
+
+Furthermore, let's visualize the computaional graph output by
+:meth:`~chainer.training.extensions.dump_graph` of
+:class:`~chainer.training.Extension` using Graphviz.
+
+::
+
+    % dot -Tpng mnist_result/cg.dot -o mnist_result/cg.png
+
+.. image:: ../../image/trainer/mnist_graph.png
+
+From the top to the bottom, you can track the data flow of the computations,
+how data and paremeters are passed to what type of :class:`~chainer.Function`
+and the calculated loss is output.
+
+8. Evaluate a pre-trained model
+'''''''''''''''''''''''''''''''
+
+.. testcode::
```

Comment:
It seems this part (or all of this page?) is intended to run in jupyter notebook.
I think it's helpful to write like `please run in jupyter notebook`.